src/components/calendar.rs

```diff
@@ -1,28 +1,263 @@
+use std::fmt::Display;
+
 use maud::{html, Markup};
+use tracing::debug;
+
+use crate::scraper::{Day, Days, MeetingTime, Section};
+
+pub struct HSL {
+    h: u16,
+    s: u8,
+    l: u8,
```

Comment:
What spec? Mozilla says they're `number` on the range [0, 360] unless you specify a unit [source](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/hsl)

Reply:
- H is a number, angle, or the keyword `none` (i'm assuming we just want number since it's implicitly degrees)
- S and L are a percentage, which is a number followed by a percent. We leave the percent out which isn't spec compiant, but who cares
- [number](https://developer.mozilla.org/en-US/docs/Web/CSS/number) is defined as an integer with optionally either decimal digits or scientific notation to denote fractional units (smaller than 1).

If you don't think this matters then it's fine which is why I put "nit", I don't think being limited to whole numbers is a big deal, but the spec does specify a larger range of values than this type allows.

As an alternative, we could just do this:

```rust
let color: String = format!("hsl({}, 50%, 25%)", sec.crn % 360);
```